agents/mutator.py

```
@@ -1,45 +1,62 @@
 from utils.logger import log
 
+
 def mutate_protocol(protocol_name):
     """Analyze protocol outcomes and mutate if failure rate is high"""
     # Try database stats first, fall back to file stats
     try:
         from utils.db_tracker import get_protocol_stats, track_mutation
+
         stats = get_protocol_stats(protocol_name)
-    except:
+    except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)